docs/ref/databases.txt

```diff
@@ -479,7 +479,9 @@ MySQL has a couple drivers that implement the Python Database API described in
 
 .. _MySQL Connector/Python: https://dev.mysql.com/downloads/connector/python/
 
-These drivers are thread-safe and provide connection pooling.
+The MySQL Connector/Python driver is thread-safe and supports connection
+pooling, whereas mysqlclient does not have built-in connection pooling
+capabilities.
```

Comment:
While `MySQL Connector/Python` supports connection pooling through `mysql.connector.pooling` [source](https://dev.mysql.com/doc/connector-python/en/connector-python-connection-pooling.html) Django itself doesn't make use of it like we do with Postgres so I wonder if its even worth pointing out. Thoughts on that @sarahboyce and @apollo13 since you worked on fad334e1a9b54ea1acb8cce02a25934c5acfe99f.

Reply:
Nice find

Reply:
Do you think its worth mentioning `mysqlclient`, or should this note be removed altogether?

Reply:
Removed in https://github.com/django/django/commit/fdae943a7cb377139c53764d950284c521947a07.